apps/right-to-rent-check/acceptance/features/landlord-agent.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const steps = require('../../');
+
+Feature('landlord agent step')
+
+Before((
+  I,
+  landlordAgentPage
+) => {
+  I.visitPage(landlordAgentPage, steps)
+});
+
+Scenario('I select Landlord, click ‘Continue’ & go to /landlord-details step', (
+  I,
+  landlordAgentPage,
+  landlordDetailsPage
+) => {
+  landlordAgentPage.selectLandlordAndSubmit();
```

Comment:
I'm being honest, I'd just do away with the helpers altogether though.

```js
I.click(landlordAgentPage.fields.landlord);
I.submitForm();
```

This is hardly complex code, and makes it wholly obvious what is going on without having to go and open another file. Adding 8 lines of helper functions to remove 3 lines of quite simple implementation looks a little like over-engineering to me.